metafile.yaml

```diff
@@ -15,30 +15,33 @@ Behavior:
       reference_frame: ''
 Ecephys:
   Device:
-    name: Device
+    name: ADDME
   ElectrodeGroup:
-    description: description
-    device: Device
+    description: ''
```

Comment:
I don't like empty strings. In the past they have caused errors on read, if a checker asks `if obj.description`, meaning to check if it is `None`, but also checking if it is an empty string. If a field as required, then we should either add it ourselves or require that a user add it.

Reply:
Understood. I changed it for ElectrodeGroup description.